immunisations/README.md

```diff
@@ -0,0 +1,145 @@
+| resource.extension.url                                      | fixed value = `https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-VaccinationProcedure-1`    |
+| resource.extension.valueExtension.url                       | fixed value = `https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid`                          |
+| resource.extension.valueExtension.valueCodeableConcept.text | `ObservationStatement.code.displayname`                                                                          |
+| resource.extension.url[1]                                   | fixed value = `https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-DateRecorded-1`            |
+| resource.extension.valueString                              | `ObservationStatement.effectiveTime.highValue`                                                                   |
+| resource.identifier.system                                  | `ObservationStatement.code.displayname`                                                                          |
+| resource.identifier.value                                   | `ObservationStatement.code.displayname`                                                                          |
+| resource.status                                             | `ObservationStatement.code.displayname`                                                                          |
+| resource.notGiven                                           | boolean value = `if passed through then set to true, otherwise it's false`                                       |
+| resource.patient.reference                                  | `ObservationStatement.code.displayname`                                                                          |
+| resource.encounter.reference                                | `ObservationStatement.code.displayname`                                                                          |
+| resource.date                                               | `ObservationStatement.code.displayname`                                                                          |
+| resource.primarySource                                      | `ObservationStatement.code.displayname`                                                                          |
+| resource.practitioner.actor.reference                       | `ObservationStatement.code.displayname`                                                                          |
+| resource.note.text                                          | `ObservationStatement.code.displayname`                                                                          |
+| resource.note.text[1]                                       | `ObservationStatement.code.displayname`                                                                          |

```

Comment:
I'd recommend using the `xPath` style syntax for the values on the right hand side.